doc/classes/Object.xml

```diff
@@ -260,9 +290,11 @@
 				[/csharp]
 				[/codeblocks]
 				While all options have the same outcome ([code]button[/code]'s [signal BaseButton.button_down] signal will be connected to [code]_on_button_down[/code]), option 3 offers the best validation: it will print a compile-time error if either the [code]button_down[/code] signal or the [code]_on_button_down[/code] callable are undefined. On the other hand, option 2 only relies on string names and will only be able to validate either names at runtime: it will print a runtime error if [code]"button_down"[/code] doesn't correspond to a signal, or if [code]"_on_button_down"[/code] is not a registered method in the object [code]self[/code]. The main reason for using options 1, 2, or 4 would be if you actually need to use strings (e.g. to connect signals programmatically based on strings read from a configuration file). Otherwise, option 3 is the recommended (and fastest) method.
-				[b]Parameter bindings and passing:[/b]
+				[b]Binding and passing parameters:[/b]
 				For legacy or language-specific reasons, there are also several ways to bind parameters to signals. One can pass a [code]binds[/code] [Array] to [method Object.connect] or [method Signal.connect], or use the recommended [method Callable.bind] method to create a new callable from an existing one, with the given parameter binds.
 				One can also pass additional parameters when emitting the signal with [method emit_signal]. The examples below show the relationship between those two types of parameters.
+				
+				While there are multiple ways to bind parameters to connections, the recommended syntax
```

Comment:
Also unfinished.

Reply:
Did you notice this PR is still a draft?

Reply:
Sure. If this is intended for now, then sorry and feel free to ignore the comments.

_These lines just seemed as a consequence of some kind of text editor bug (?), or whatever._

Reply:
> These lines just seemed as a consequence of some kind of text editor bug (?), or whatever.

In reality they're dummy lines, partial attempts to rewrite the same thing in a different way to see what is best. I have not removed and tidied up a few just yet.